automation/taskcluster/decision_task_release.py

```diff
@@ -29,15 +31,26 @@
 )
 
 
-def load_artifacts_manifest():
-    return yaml.safe_load(open('automation/taskcluster/artifacts.yml', 'r'))
+def fetch_artifact_metadata():
```

Comment:
Nit (not blocking): maybe it's worth extracting L35-40 into a function somewhere in an `utils.py` module or alike and reuse in both the decision tasks? (https://github.com/mozilla-mobile/android-components/pull/1284/files#diff-168d551d2c0830884a5caeb50ae576e9R29 too)